### PR TITLE
updated readme file with helpful samples, resources

### PR DESCRIPTION
**Pull request by garora**
@Eilon - as discussed, here is a fresh/clean PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,18 @@ ASP.NET Web Hooks provide support for sending and receiving WebHooks.
 Please see the blog [Introducing Microsoft ASP.NET WebHooks Preview](http://blogs.msdn.com/b/webdev/archive/2015/09/04/introducing-microsoft-asp-net-webhooks-preview.aspx) 
 for an introduction to Microsoft ASP.NET WebHooks.
 
-Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
\ No newline at end of file
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+##Helpful samples and resources
+Following(s) are helpful resources from our contributors:
```
Comment (Eilon) [inline README.md]: Can remove this line.

File: README.md [hunk 2]
```diff
@@ -5,4 +5,18 @@ ASP.NET Web Hooks provide support for sending and receiving WebHooks.
...
 
-Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
\ No newline at end of file
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+##Helpful samples and resources
+Following(s) are helpful resources from our contributors: 
+
+###Samples
+- [GitHub](/samples/#)
```
Comment (Eilon) [inline README.md]: I think I suggested to change this to say just "All samples" (and remove the `#` from the URL).

Comment (garora) [inline README.md]: My bad! I missed that message. I am pushing updated PR.
Thanks for your support.

**Comment by Eilon** [comment]
Thanks! Can you squash your commits into one? Then we can merge.

**Comment by garora** [comment]
Ok will, if it work. If not, will make fresh PR :)

**Comment by Eilon** [comment]
Should be easy :smile:

**Comment by garora** [comment]
Awesome! Let me give it try.
Last time we noticed few bad things....

**Comment by garora** [comment]
Unfortunately, Squash did not work for me, with this I've to make force push my repository, which I do not like at the moment. Making a fresh/clean PR.

**Pull request closed without merging**